website/package.json

```diff
@@ -84,9 +84,10 @@
         "eslint-plugin-prefer-arrow": "^1.2.3",
         "eslint-plugin-react": "^7.35.2",
         "eslint-plugin-react-hooks": "^4.6.2",
+        "happy-dom": "^15.7.4",
```

Comment:
What's the benefit of using happy-dom?

Reply:
It provides a different `File` implementation that works with the newer MSW version. This was suggested upstream here: https://github.com/mswjs/msw/issues/2273#issuecomment-2351508014

Reply:
MSW doesn't work with jsdom, see issue